src/clarity-angular/wizard/wizard-page.ts

```diff
@@ -171,12 +171,12 @@ export class WizardPage implements OnInit {
 
     // If our host has an ID attribute, we use this instead of our index.
     @Input("id")
-    _id: string = (wizardPageIndex++).toString();
+    _id: string = `${wizardPageIndex++}`;
 
     public get id() {
-        if (!this._id) {
+        if (this._id == null) {
```

Comment:
- It allows compare to `null` with double equals by `allow-null-check` flag.
- I thought AoT would perform type guard through @Input() decorator, turns out still not there yet, therefore it's going to require runtime type checking against every non-primitive type as well.